client/src/components/NavBar/theme.ts

```diff
@@ -0,0 +1,11 @@
+export const PT_GRID_SIZE_PX = 8;
+
+export const PT_TEXT_COLOR = "#10161A";
```

Comment:
Please use the Figma file as the reference for styling rather than the existing navigation bar. We should update the existing navigation bar to match this implementation, if necessary.

All text styles and colors should reference the Single Cell theme file at `frontend/src/common/theme.ts`. You can see the corresponding variables in Figma (e.g., `Gray/300` or `fontBodyS/600`)